.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,150 @@
+{
+  "name": "mne-dev",
+  // "image": "mcr.microsoft.com/devcontainers/base:debian-11",
+  "build": { "dockerfile": "Dockerfile" },
+  "containerEnv": {
+    "PYTHONNOUSERSITE": "true", // Make Python ignore the user's site-packages folder if it exists
+    "XDG_RUNTIME_DIR": "/home/vscode/.cache/xdgr"
+  },
+  "features": {
+    // See https://containers.dev/features
+    // APT packages
+    "ghcr.io/rocker-org/devcontainer-features/apt-packages:1": {
+      "packages": "mesa-utils,libegl1,^libxcb.*-dev,libx11-xcb-dev,libglu1-mesa-dev,libxrender-dev,libxi-dev,libxkbcommon-dev,libxkbcommon-x11-dev"
+    },
+    // Git
+    "ghcr.io/devcontainers/features/git:1": {
+      "version": "os-provided",
+      "ppa": "true"
```

Comment:
curious why installing git from a PPA here. Are there very new features that are needed, that aren't in the APT version?

Reply:
Good catch, this seems to be some cruft from earlier testing. I believe this can be safely changed to `false`